uint8 MType
uint8 SupType
float32 PosX
float32 PosY
float32 PosZ
uint8 FilterType
